packages/contracts-bedrock/test/kontrol/README.md

```diff
@@ -1,98 +1,129 @@
 
-- [`KontrolDeployment.sol`](./deployment/KontrolDeployment.sol): Simplified deployment sequence for Kontrol proofs
-- [`DeploymentSummary.t.sol`](./deployment/DeploymentSummary.t.sol): Tests for the summarization of custom deployment
+<pre>
+├── <a href="./pausability-lemmas.k">pausability-lemmas.k</a>: File containing the necessary lemmas for this project
+├── <a href="./deployment">deployment</a>: Custom deploy sequence for Kontrol proofs and tests for its <a href="https://github.com/runtimeverification/kontrol/pull/271">fast summarization</a>
+│   ├── <a href="./deployment/KontrolDeployment.sol">KontrolDeployment.sol</a>: Simplified deployment sequence for Kontrol proofs
+│   └── <a href="./deployment/DeploymentSummary.t.sol">DeploymentSummary.t.sol</a>: Tests for the summarization of custom deployment
+├── <a href="./proofs">proofs</a>: Where the proofs (tests) themselves live
+│   ├── *.k.sol</a>: Symbolic property tests for contracts
+│   ├── <a href="./proofs/interfaces">interfaces</a>: Interface files for src contracts, to avoid unnecessary compilation of contracts
+│   └── <a href="./proofs/utils">utils</a>: Proof dependencies, including the autogenerated deployment summary contracts
+└── <a href="./scripts">scripts</a>: Where the scripts of the projects live
+    ├── <a href="./scripts/json">json</a>: Data cleaning scripts for the output of <a href="./KontrolDeployment.sol">KontrolDeployment.sol</a>
+    ├── <a href="./scripts/make-summary-deployment.sh">make-summary-deployment.sh</a>: Executes <a href="./KontrolDeployment.sol">KontrolDeployment.sol</a>, curates the result and writes the summary deployment contract
+    └── <a href="./scripts/run-kontrol.sh">run-kontrol.sh</a>: Wrapper around the kontrol CLI to run the proofs
+</pre>
 
-### [`proofs`](./proofs) folder
+### Installation
 
-- [`L1CrossDomainMessenger.k.sol`](./proofs/L1CrossDomainMessenger.k.sol): Symbolic property tests for [`L1CrossDomainMessenger`](../../src/L1/L1CrossDomainMessenger.sol)
-- [`L1ERC721Bridge.k.sol`](./proofs/L1ERC721Bridge.k.sol): Symbolic property tests for [`L1ERC721Bridge`](../../src/L1/L1ERC721Bridge.sol)
-- [`L1StandardBridge.k.sol`](./proofs/L1StandardBridge.k.sol): Symbolic property tests for [`L1StandardBridge`](../../src/L1/L1StandardBridge.sol)
-- [`OptimismPortal.k.sol`](./proofs/OptimismPortal.k.sol): Symbolic property tests for [`OptimismPortal`](../../src/L1/OptimismPortal.sol)
-- [`interfaces`](./proofs/interfaces): Files with the signature of the functions involved in the verification effort
-- [`utils`](./proofs/utils): Proof dependencies, including the summary contracts
+1. `cd` to the root of this repo.
+2. Install Foundry by running `pnpm install:foundry`. This installs `foundryup`, the foundry toolchain installer, then installs the required foundry version.
+3. Install Kontrol by running `pnpm install:kontrol`. This installs `kup`, the package manager for RV tools, then installs the required kontrol version.
```

Comment:
Would it be good to add at the end something like 
"Alternatively, run `curl -L https://foundry.paradigm.xyz | bash`" for Foudry and 
"Alternatively, run `bash <(curl https://kframework.org/install)` and `kup install kontrol --version v$(jq -r .kontrol < versions.json)`" for Kontrol? Perhaps not, but I thought I'd ask

Reply:
Going to leave this as-is :)